xc/device.ml

```diff
@@ -2549,13 +2549,7 @@ module Backend = struct
                   List.map (fun x -> ["-qmp"; sprintf "unix:/var/run/xen/qmp-%s-%d,server,nowait" x domid]) |>
                   List.concat in
 
-        let has_platform_device =
```

Comment:
I think it was agreed that it's safe to drop for qemu-upstream-uefi only. We still didn't clarify the usecase for it in compat mode.